cmake/oneDPLWindowsIntelLLVMApply.cmake

```diff
@@ -37,5 +39,5 @@ endif()
 if (INTELLLVM_WIN_STD_IGNORE_FIX)
     include(Compiler/CMakeCommonCompilerMacros)
     # Setting c++ standard default to 14, and enabling usage of CMAKE_CXX_STANDARD
-    __compiler_check_default_language_standard(CXX 2020 14)
+    __compiler_check_default_language_standard(CXX 2023 14)
 endif()
```

Comment:
The comment above the line does not describe what the arguments mean. @danhoeflinger, can you improve the comment so that the meaning is clear and the intent of the change can be verified?

Reply:
This cmake file is a "workaround" for specific versions of CMake on windows, fixing some bugs in their setup of the intelLLVM compiler on windows.
The `2020` here corresponds to the minimum version of the intel LLVM compiler, for which the default of c++14 is used. I believe this line of code should not change in this PR.

Please find my suggestion for a change to the comment to make it more clear above. If you would prefer I create a separate PR for this comment fix for you to then rebase from, let me know.

Reply:
I realize my statement above might be ambiguous or unclear. I think this specific line should be reverted back to the state it was in main, as there is no reason to change the minimum intelLLVM version from 2020 to 2023. I think you understood but I want to make sure.

Reply:
I understood, yes. I'm keeping this comment open exactly to have this change reverted.

Reply:
applied, change reverted